TUTORIAL.md

```
@@ -202,6 +203,29 @@ session = osidb_bindings.new_session(
+By default, the session identifies itself as `osidb-bindings/{version} (python-requests/{version})`. You can pass a custom `user_agent` to make your application identifiable in OSIDB logs:
+
+```python
+session = osidb_bindings.new_session(
+    osidb_server_uri="http://localhost:8000/",
+    user_agent="MyApp v1.0.0 (python-requests/2.32.3) prod",
+)
+```
+
+If the provided user-agent follows the `{APP_NAME} v{VERSION} ({library/version}) {ENV}` format, the library part is automatically replaced with the osidb-bindings identity:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)